config/fcos/v1_6_exp/validate_test.go

```diff
@@ -231,3 +231,76 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path:           util.StrToPtr("/foo"),
+								WipeFilesystem: util.BoolToPtr(false),
```

Comment:
Generally try and make your configs as lean as possible to be explicit about what you are testing. 
ie. `WipeFilesystem`

Reply:
not sure why i kept that in, have it fixed now!